etna/.npm-packages/lib/node_modules/etna-js

```diff
@@ -0,0 +1 @@
+/etna/packages/etna-js
```

Comment:
Hm, this file isn't tracked for me. Do we need to add it to the repo, or is it created by one of the docker build scripts?

Reply:
Hmm, I didn't add this, so I assume some build process must be creating it.

Reply:
Huh.  I hadn't seen that before, to be honest.  I don't know how this got creatd.